cmd/endpoint.go

```diff
@@ -133,6 +133,11 @@ func NewEndpoint(arg string) (ep Endpoint, e error) {
 		}
 	} else {
 		u = &url.URL{Path: path.Clean(arg)}
+		// Check if the arg is a localhost or an IP without a scheme, then ask user to start with a scheme.
+		if strings.Contains(arg, "localhost") || checkIPv4(arg) {
+			return ep, fmt.Errorf("invalid URL endpoint format: missing scheme http or https")
+		}
+		// Only set isLocal to true after getting a valid localhost or IP address with the scheme.
```

Comment:
This comment is no meaning to this context i.e. this section of the code knows given `arg` is a path.  You could remove this.